vizro-core/tests/e2e/vizro/dashboards/yaml/dashboard.yaml

```
@@ -0,0 +1,290 @@
+                type: container
+                title: filters components container
+                components:
+                  - figure:
+                      _target_: scatter
+                      color: sepal_width
+                      data_frame: iris
+                      x: sepal_length
+                      y: petal_width
+                    id: scatter
+                    type: graph
+                  - text: |
+                      ![icon-top](assets/images/icons/content/features.svg)
+
+                      Leads to the home page on click.
+                    type: card
+                    href: /
+                  - type: button
+                    text: Export data
+                    actions:
+                      - function:
+                          _target_: export_data
+                          targets: [scatter]
+                          file_format: csv
+                      - function:
+                          _target_: export_data
+                          targets: [scatter]
+                          file_format: xlsx
+                layout:
+                  grid: [[0, 1], [0, 1], [0, 2]]
+      - figure:
+          _target_: box
+          color: sepal_width
+          data_frame: iris
+          x: sepal_length
+          y: petal_width
+        id: box graph
+        type: graph
+    controls:
+      - column: species
+        targets:
+          - scatter
+          - box graph
+        type: filter
+        selector:
+          type: dropdown
+          id: drop-filters-page
+      - column: species
+        targets:
+          - scatter
+          - box graph
+        type: filter
+        selector:
+          type: radio_items
+          options: [setosa, versicolor, virginica]
+          id: radio-filters-page
+      - column: species
+        targets:
+          - scatter
+        type: filter
+        selector:
+          type: checklist
+          options: [setosa, versicolor, virginica]
+          id: check filters-page
+      - column: petal_width
+        targets:
+          - scatter
+        type: filter
+        selector:
+          type: slider
+          step: 0.5
+          id: slider-filters-page
+      - column: sepal_length
+        targets:
+          - scatter
+          - box graph
+        type: filter
+        selector:
+          type: range_slider
+          step: 1.0
+          id: range slider-filters-page
+    title: filters page (tabs + containers)
+    path: /filters-page-tabs--containers
+  - components:
+      - type: tabs
+        tabs:
+          - id: parameters-tab_container
+            type: container
+            title: parameters-tab_container
+            components:
+              - type: tabs
+                id: sub tab params
+                tabs:
+                  - id: parameters sub t@b container 1
+                    type: container
+                    title: parameters sub t@b container 1
```

I am seeing more **@** signs instead of an **a** in this file - did you do it on purpose to test something? 🤔 

```suggestion
                  - id: parameters sub tab container 1
                    type: container
                    title: parameters sub tab container 1
```

---

Same thing as in previous comment, just a full copy of the default dashboard

---

If there's no need for testing the @, then we can "fix" it and revert the "a" instead for both default and yaml tests.